### ISSUE TITLE
An Exact Mass search condition

### ISSUE THREAD

**Issue by xieyying** [mSorok/NaturalProductsOnline#124]
Can an Exact Mass search condition be added to the search page?  Because Exact Mass is more important for dereplication of natural products than molecular weight.

**Comment by samuelbertrand44** [comment]
I agree, this is very limiting that accurate mass is not searchable. can this be added ?
do you need help ?
